dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESClient.java

```diff
@@ -226,15 +231,13 @@ private Server setUpHttpConf(Server currentServer, final String serverId,
                             .toString()
                             : ClusterFactory.getNextAvailablePort(serverId, ServerPort.ES_HTTP_PORT,
                                     externalSettings);
-            currentServer = Server.builder(currentServer).withEsHttpPort(Integer.parseInt(httpPort))
-                    .build();
+
```

Comment:
Idea is to leave it with single responsability, which is just set the http conf to external settings, and don't alter the server in addition to that, which will happen later in a different method